### PR TITLE
Add second TA Rubrics pilot without AI

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Adds a new pilot called `non-ai-rubrics` that allows users to see our new rubrics without any AI evaluation.

Screenshot of how this looks for the user (icon provided by Molly):

Screenshot 2023-10-31 at 9 42 46 AM

The AI evaluation is [gated on](https://github.com/code-dot-org/code-dot-org/blob/8f06a1bcd1e2896804e4682e2b698da6c036a084/dashboard/app/controllers/activities_controller.rb#L53) being in the `ai-rubrics` pilot, so this will not automatically run AI evaluations for these users.

### CODE REVIEW

File: apps/src/templates/rubrics/RubricContainer.jsx [hunk 1]
```diff
@@ -31,9 +31,6 @@ export default function RubricContainer({
   const canProvideFeedback = !!studentLevelInfo && onLevelForEvaluation;
 
   const [selectedTab, setSelectedTab] = useState(TAB_NAMES.RUBRIC);
-  const [teacherHasEnabledAi, setTeacherHasEnabledAi] = useState(
```
Comment (bethanyaconnor) [inline apps/src/templates/rubrics/RubricContainer.jsx:34]: When I originally created `RubricSettings`, I had added a toggle to run AI eval on and off. I removed that toggle but didn't remove this unnecessary state. Let me know if I should break this out into a separate PR.

File: apps/src/templates/rubrics/RubricFloatingActionButton.jsx [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import React, {useEffect, useMemo, useState} from 'react';
 import PropTypes from 'prop-types';
 import style from './rubrics.module.scss';
-const icon = require('@cdo/static/ai-fab-background.png');
+import aiFabIcon from '@cdo/static/ai-fab-background.png';
+import rubricFabIcon from '@cdo/static/rubric-fab-background.png';
```
Comment (bethanyaconnor) [inline apps/src/templates/rubrics/RubricFloatingActionButton.jsx:5]: @snickell -- fyi these will cause conflicts with your PR

File: apps/src/templates/rubrics/RubricContainer.jsx [hunk 3]
```diff
@@ -65,6 +62,11 @@ export default function RubricContainer({
     fetchAiEvaluations();
   }, [fetchAiEvaluations]);
 
+  // Currently the settings tab only provides a way to manually run AI.
+  // In the future, we should update or remove this conditional when we
+  // add more functionality to the settings tab.
```
Comment (davidsbailey) [inline apps/src/templates/rubrics/RubricContainer.jsx:67]: 💯

File: apps/src/templates/rubrics/RubricFloatingActionButton.jsx [hunk 4]
```diff
@@ -55,6 +57,8 @@ export default function RubricFloatingActionButton({
     }
   }, [eventData, studentLevelInfo]); // Neither of these should change, so this should run once
 
+  const icon = aiEnabled ? aiFabIcon : rubricFabIcon;
```
Comment (davidsbailey) [inline apps/src/templates/rubrics/RubricFloatingActionButton.jsx:60]: looks good for this PR, but wondering @tess323 if we also want to show the rubric-only icon on lesson 7, where we will have a rubric but no ai yet? unsure if this will be a common case for us

File: apps/src/templates/rubrics/RubricContainer.jsx (hunk 1 above)
Comment (davidsbailey) [reply apps/src/templates/rubrics/RubricContainer.jsx:34]: this seems fine to include given the size of the PR. thanks for catching this

**Review by davidsbailey — APPROVED** [review]
looks great, thank you Bethany!
